app/frontend/src/test/serviceMockDefaults.ts

```diff
@@ -1,11 +1,12 @@
+import { User } from "pb/api_pb";
 import { HostRequestStatus } from "pb/conversations_pb";
 import messages from "test/fixtures/messages.json";
 import users from "test/fixtures/users.json";
 
 const userMap = new Map(users.map((user) => [user.userId, user]));
 
 export async function getUser(userId: string) {
-  return userMap.get(+userId);
+  return userMap.get(+userId) as User.AsObject;
```

Comment:
Why this change?

Reply:
It's because I've used the real return type of `getUser` and the fixtures don't actually have the full user objects (a lot of missing keys deemed required in the type). Maybe I'll just add in everything as in the default user and stop being lazy 😅  I imagine eventually once we have better test coverage, we'll need those properties on the user anyway. Since it's already in a separate JSON file, I don't see it being a problem (mainly bloating up the test file with properties you don't care about in a specific test suite if it were inlined)

Reply:
Oh crap actually, might still need this or a non-null assertion since `map.get` returns `User.AsObject | undefined` but it should never be undefined in our case as we should always be using known userIds in our tests.